src/content/configuration/entry-context.md

```diff
@@ -57,6 +58,58 @@ module.exports = {
 
 If a string or array of strings is passed, the chunk is named `main`. If an object is passed, each key is the name of a chunk, and the value describes the entry point for the chunk.
 
+### Entry descriptor
+
+If an object is passed the value might be a string, array of strings or descriptor:
+
+```js
+module.exports = {
+  //...
+  entry: {
+    home: './home.js',
+    about: ['./about.js'],
+    contact: { import: './contact.js' }
+  }
+};
+```
+
+Descriptor syntax might be used to pass additional options to an entry point.
+
+
+### Output filename
+
+By default, an output filename for the entry chunk is getting from [`output.filename`](/configuration/output/#outputfilename) but you can specify a custom output filename for a specific entry:
+
+```js
+module.exports = {
+  //...
+  entry: {
+    app: './app.js',
+    home: { import: './contact.js', filename: 'pages/[name][ext]' },
+    about: { import: './about.js', filename: 'pages/[name][ext]' }
+  }
+};
+```
+
+Descriptor syntax was used here to pass `filename`-option to the specific entry points.
```

Comment:
We definitely need to show an example showing all the keys that descriptors receive.

Reply:
I’ve already did it in filename and deprndOn section. Or do you mean some example with all the fields in one place?

Reply:
> Or do you mean some example with all the fields in one place?

Yes, that would be great.